### PR TITLE
FEAT: Add aws_anonymous profile

### PR DESCRIPTION
**Pull request by maxulysse**
to simplify --no-sign-request usage with aws cli

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
How about adding this to awsbatch config directly? It should solve the issues with igenomes for everyone, right?

**Comment by maxulysse** [comment]
not sure we want that actually.
People might already have solved that on their side.
This is just to make it work for anyone wherever.

File: docs/aws_anonymous.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# nf-core/configs: AWS anonymous
+
+To be used when one want the aws cli to specify `--no-sign-request`.
```
Comment (jfy133) [inline docs/aws_anonymous.md:3]:
```suggestion
To be used when one wants the aws cli to specify `--no-sign-request`.
```

**Review by jfy133 — APPROVED** [review]
Missing adding to README (maybe under a special section), but otherwise LGTM

**Comment by maxulysse** [comment]
> Missing adding to README (maybe under a special section), but otherwise LGTM

for me mainly missing if we actually want to do that or not...

**Comment by ewels** [comment]
I have a feeling that this will break people's AWS batch workflows, as sign requests are needed to access most user-specific resources (legitimately). This will disable all sign requests for anything in the workflow.. It'd be good if someone could confirm?

See related: https://github.com/nextflow-io/nextflow/issues/4732

**Comment by maxulysse** [comment]
> I have a feeling that this will break people's AWS batch workflows, as sign requests are needed to access most user-specific resources (legitimately). This will disable all sign requests for anything in the workflow.. It'd be good if someone could confirm?
> 
> See related: [nextflow-io/nextflow#4732](https://github.com/nextflow-io/nextflow/issues/4732)

Yeah, I'm not fully happy with it, which is why I haven't merged it now, I know it can solve some issue for some people, but I'm afraid it'll break stuff for other people.

**Comment by jfy133** [comment]
@maxulysse converting to draft. As you said relatively recently you're still unsure but could be relevant I'll leave it here for now. Please otherwise merge/close as necessary

**Comment by maxulysse** [comment]
Let's just close it

**Pull request closed without merging**